pkg/kubelet/kubelet_volumes.go

```diff
@@ -51,12 +51,24 @@ func (kl *Kubelet) ListVolumesForPod(podUID types.UID) (map[string]volume.Volume
 
 // podVolumesExist checks with the volume manager and returns true any of the
 // pods for the specified volume are mounted.
-func (kl *Kubelet) podVolumesExist(podUID types.UID) bool {
+func (kl *Kubelet) podVolumesExist(podUID types.UID, checkVolumePaths bool) bool {
```

Comment:
I think it might be more clear to  create another function for this purpose instead of reuse this one and adding a parameter of it. They are calling different functions. kl.getPodVolumePathListFromDisk , kl.getMountedVolumePathListFromDisk

Reply:
I added another function for the `podVolumePathsExist`.